pgdrive/utils/basic_utils.py

```diff
@@ -1,11 +1,15 @@
 import math
```

Comment:
so why this file names "basic_utils"? Should we change it to "scene_utils" or "localization_utils" or so?

Reply:
Yes, it is better to name it to scene_utils